sunpy/net/base_client.py

```diff
@@ -202,10 +202,14 @@ def _print_client(client, html=False):
     lines = [class_name, dedent(client.__doc__.partition("\n\n")[0])]
     if html:
         lines = [f"<p>{line}</p>" for line in lines]
-    lines.extend(t.pformat_all(show_dtype=False, max_width=width, align="<", html=html))
+    if visible_entries is not None:
+        lines.extend(t.pformat_all(max_lines=visible_entries, show_dtype=False, max_width=width, align="<", html=html))
+    else:
+        lines.extend(t.pformat_all(show_dtype=False, max_width=width, align="<", html=html))
```

Comment:
it's perfectly allowed for max_lines to be None, so this can just be:
```suggestion
   lines.extend(t.pformat_all(max_lines=visible_entries, show_dtype=False, max_width=width, align="<", html=html))
```